samples/geo-ip/lib/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "geoip-component"
```

Comment:
...speaking of which, bikeshedding: `geoip-static-db`? Really doesn't matter, but I do think it should match the directory.